packages/verkle/src/node/types.ts

```diff
@@ -45,3 +51,15 @@ export interface VerkleNodeOptions {
 }
 
 export const NODE_WIDTH = 256
+
+export const createZeroLeaf = () => new Uint8Array(32)
+
+export const createDeletedLeafValue = () =>
+  setLengthLeft(bigIntToBytes(bytesToBigInt(new Uint8Array(32)) + BigInt(2 ** 128)), 32)
+
+export const DELETED_LEAF_VALUE = setLengthLeft(
+  bigIntToBytes(bytesToBigInt(new Uint8Array(32)) + BigInt(2 ** 128)),
+  32
+)
```

Comment:
Why do we need a constant in addition to a function? Also, could that be made more efficient by manipulating bytes directly rather than doing BigInt operations and multiple conversions?

Reply:
I did the constant since we compare the bytes from a serialized node that's stored in the db to the constant when reinstating it. I guess we might not need to now that you mention it though. Just need a clean way to represent the deleted value type in the db.

Reply:
And yes, doing this with byte arithmetic is the more efficient way. Just hadn't quit decided on the best way to do it. The more I think about it, the spec is a little goofy here. Seems like there should be a more efficient way to represent deleted value

Reply:
And looks like you already took care of it. I like your edits.